test/components/FileInput.spec.js

```diff
@@ -144,4 +150,32 @@ describe('components', () => {
       expect(tree).toMatchSnapshot();
     });
   });
+
+  describe('Metadata', () => {
```

Comment:
It is a semantic thing, but in this section, you're not _describing_ the `Metadata`. Still, you're _describing_ the `FileInput` component behaviour (and don't get me wrong, the tests you wrote are meaningful and must be here).

There's not much to test in the `Metadata` component, but a Snapshot testing is also helpful. It will help other developers `to see` what you're really doing there, and if someone modifies it she/he will know it (because tests will break).